packages/active-campaign-client/src/handlers/addContact.ts

```diff
@@ -0,0 +1,51 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { acClient } from '../activeCampaignClient';
+import { SignUpUserData } from 'nextjs-website/src/lib/types/sign-up';
+import { ContactPayload } from '../types/contactPayload';
+
+export type HandlerEvent = Pick<APIGatewayProxyEvent, 'body'>;
+
+export async function handler(
+  event: HandlerEvent
+): Promise<APIGatewayProxyResult> {
```

Comment:
Given that we don't actually have an Api Gateway in front of the lambda, is the class used only for convenience, or may be replaced by the class based on the actual caller? (maybe sqs?)